### PR TITLE
fix mismatched symbols

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: src/doc/rustdoc/src/write-documentation/documentation-tests.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ only shows the part you care about.
 
 ```rust
 /// ```ignore
-/// fn foo() {
+/// fn foo() {}
```
Comment (Nilstrieb) [inline src/doc/rustdoc/src/write-documentation/documentation-tests.md:320]: this is probably on purpose, being invalid rust code

**Comment by cuishuang** [comment]
Thanks. 
Done

File: src/doc/rustdoc/src/write-documentation/documentation-tests.md (hunk 1 above)
Comment (cuishuang) [reply src/doc/rustdoc/src/write-documentation/documentation-tests.md]: Thanks~

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors r+ rollup

**Pull request merged**